### ISSUE TITLE
Example links cut off 

### ISSUE THREAD

**Issue by Antonialock** [pombase/website#839]
On a small display, the example links 
advanced search * GO process overview * Genome browser

are cut off to the point where you cant see the genome browser link

<img width="540" alt="morga" src="https://user-images.githubusercontent.com/12727506/42308813-a0df4f42-802e-11e8-9ae5-04425dd10b3e.png">

**Comment by ValWood** [comment]
It might be nice to knock off a few of the Smart phone tickets soon too, since so many people now seem to use PomBAse on smart phones. 

This will be a nice thing to announce when the glitches are fixed.... news item  front page  feature, twitter, pombelist)

...but we don't need to announce individual fixes. We'll just tell people improvements have been made once you are happy

**Comment by kimrutherford** [comment]
I've improved the display on small screens.

> It might be nice to knock off a few of the Smart phone tickets soon too,

I've been slowly chipping away at them.  There are still quite a lot though.

If there are any that are higher priority, let me know.

**Comment by ValWood** [comment]
@Antonialock  could you look through these and flag the ones that should be done before we announce.
If they are only cosmetic we can make them lower priority

**Comment by kimrutherford** [comment]
> It seems you have done them all. The 2 remaining are low priority.

Now down to one (which I can't fix).  It was easier to do them at the same time because they're related.

**Comment by ValWood** [comment]
yeah we aren't so worried about videos....

**Comment by Antonialock** [comment]
I've had a quick look around the website on my phone (I have a very large screen) and it looks fine.
